Formula/openshift-cli.rb

```diff
@@ -1,15 +1,14 @@
 class OpenshiftCli < Formula
   desc "OpenShift command-line interface tools"
   homepage "https://www.openshift.com/"
-  url "https://github.com/openshift/oc.git",
-      tag:      "openshift-clients-4.12.0-202208031327",
-      revision: "3c85519af6c4979c02ebb1886f45b366bbccbf55"
+  url "https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.12.13/openshift-client-src.tar.gz"
```

Comment:
Channels are documented here: https://docs.openshift.com/container-platform/latest/updating/understanding-upgrade-channels-release.html

The reason I'm using 4.12.13 as opposed to 4.12.10 or 4.12.12 is because 4.12.13 is the first version with the source tarball in this format to allow for homebrew to consume it in a stable manner. However, it is expected that 4.12.13 will become the stable version in a few weeks as it is promoted first to fast channel, and then to stable.

Reply:
@cho-m An update: 4.12.13 is now the stable version so the warnings from livecheck shouldn't appear anymore